src/gen.cpp

```diff
@@ -71,11 +80,22 @@ struct element {
  double T, mub, muq, mus ;
  double pi[10] ;
  double Pi ;
+ // All derivatives of the 4-velocity, following the structure
+ // dt_u0, dt_ux, dt_uy, dt_uz, dx_u0, dx_ux, dx_uy, dx_uz, dy_u0 ...
+ double u_derivatives[16] ;
```

Comment:
I wonder if it is with respect to RAM wise to always declare 16 doubles per element even if they are not used.